docs/src/extend/custom-rules.md

```diff
@@ -145,7 +145,8 @@ Additionally, the `context` object has the following methods:
 
 The main method you'll use when writing custom rules is `context.report()`, which publishes a warning or error (depending on the configuration being used). This method accepts a single argument, which is an object containing the following properties:
 
-* `message`: (`string`) The problem message.
+* `message`: (`string`) The problem message (alternative to `messageId`).
+* `messageId`: (`string`) The ID of the message (see [messageIds](#messageids)) (recommended alternative to `message`).
```

Comment:
[Suggestion] Maybe put `messageId` first? That way folks already have it in mind when they come across `message`.

```suggestion
* `messageId`: (`string`) The ID of the message (see [messageIds](#messageids)) (recommended over `message`).
* `message`: (`string`) The problem message (alternative to `messageId`).
```